packages/fxa-auth-server/lib/backendService.js

```diff
@@ -217,6 +224,12 @@ module.exports = function createBackendServiceAPI(
         payload,
         this._headers
       );
+
+      // The statsD dependency is optional
+      if (statsd) {
+        statsd.timing(`${serviceName}.${methodName}`, Date.now() - startTime);
```

Comment:
Haven't run this code yet, but does this fail gracefully when the statsd server can't be reached?

Reply:
Yeah, I didn't see anything in the log.